__tests__/app.test.js

```diff
@@ -258,3 +258,77 @@ describe("POST /api/reviews/:review_id/comments", () => {
       });
   });
 });
+describe("PATCH /api/reviews/:review_id", () => {
+  it("should return 201 with the updated review object", () => {
+    return request(app)
+      .patch("/api/reviews/2")
+      .send({ inc_votes: 2 })
+      .expect(201)
+      .then(({ body }) => {
+        const review = body.review;
+        expect(review).toHaveProperty("review_id", 2);
+        expect(review).toHaveProperty("title", "Jenga");
+        expect(review).toHaveProperty("designer", "Leslie Scott");
+        expect(review).toHaveProperty(
+          "review_img_url",
+          "https://images.pexels.com/photos/4473494/pexels-photo-4473494.jpeg?w=700&h=700"
+        );
+        expect(review).toHaveProperty(
+          "review_body",
+          "Fiddly fun for all the family"
+        );
+        expect(review).toHaveProperty("category", "dexterity");
+        expect(review).toHaveProperty("created_at", "2021-01-18T10:01:41.251Z");
+        expect(review).toHaveProperty("votes", 7);
+      });
+  });
+  it("should update the database", () => {
+    return request(app)
+      .patch("/api/reviews/2")
+      .send({ inc_votes: 2 })
+      .then(() => {
+        return db.query("SELECT * FROM reviews WHERE review_id = 2");
+      })
+      .then(({ rows }) => {
+        expect(rows[0].votes).toBe(7);
+      });
+  });
```

Comment:
This test is already done in the above test, as votes would only be 7 after the increase was made.